feature_creation_scripts/ismip6_antarctic_ocean_regions/ismip6_antarctic_ocean_regions.py

```diff
@@ -0,0 +1,279 @@
+
+    p = paths[iLongest]
+    v = p.vertices
+    x = v[:, 0]
+    y = v[:, 1]
+
+    # Antarctic stereographic
+    inProj = pyproj.Proj(init='epsg:3031')
+    # lon/lat
+    outProj = pyproj.Proj(init='epsg:4326')
+
+    poly = shapely.geometry.Polygon([(i[0], i[1]) for i in zip(x, y)])
+
+    epsilon = 1e-14
+    minY = numpy.amin(y)
+    wedge = shapely.geometry.Polygon([(epsilon, minY),
+                                      (epsilon**2, -epsilon),
+                                      (0, epsilon),
+                                      (-epsilon**2, -epsilon),
+                                      (-epsilon, minY),
+                                      (epsilon, minY)])
+
+    difference = poly.difference(wedge)
+
+    difference = shapely.ops.transform(stereo_to_lon_lat, difference)
+
+    fc = FeatureCollection()
+
+    geometry = shapely.geometry.mapping(difference)
+    # get rid of the wedge again by rounding the coordinates
+    geometry['coordinates'] = _round_coords(geometry['coordinates'])
+
+    fc.add_feature(
+        {"type": "Feature",
+         "properties": {"name": "Contour {}".format(contourValue),
+                        "author": author,
+                        "object": 'region',
+                        "component": 'ocean'},
+         "geometry": geometry})
+
+    return fc
+
+
+def make_polygon(lons, lats, name, author, tags):
+    fc = FeatureCollection()
+
+    coords = list()
+    for index in range(len(lons)):
+        coords.append([lons[index], lats[index]])
+    coords.append([lons[0], lats[0]])
+
+    fc.add_feature(
+        {"type": "Feature",
+         "properties": {"name": name,
+                        "author": author,
+                        "object": 'region',
+                        "component": 'ocean',
+                        "tags": tags,
+                        "zmin": -1000.,
+                        "zmax": -400.},
+         "geometry": {
+             "type": "Polygon",
+             "coordinates": [coords]}})
+    return fc
+
+
+def shelf_polygon(lons, lats, name, author, tags, fcContour):
+    fc = make_polygon(lons, lats, name, author, tags)
+
+    lons = [-180., -180., 180., 180.]
+    lats = [-90., 90., 90., -90.]
+    fc_world = make_polygon(lons, lats, name, author, tags)
+
+    fcContour = fc_world.difference(fcContour)
+
+    fcShelf = fc.difference(fcContour)
+
+    props = fcShelf.features[0]['properties']
+    props['name'] = props['name'] + ' Shelf'
+    props['tags'] = props['tags'] + ';Shelf'
+    props['zmin'] = -1000.
+    props['zmax'] = -200.
+
+    return fcShelf
+
+
+def main():
+    author = 'Xylar Asay-Davis, Alice Barthel, Nicolas Jourdain'
+    tags = 'Antarctic;ISMIP6'
+
+    # make a geometric features object that knows about the geometric data
+    # cache up a couple of directories
+    gf = GeometricFeatures('../../geometric_data')
+
+    bedmap2_bin_to_netcdf('bedmap2.nc')
+
+    fcContour1500 = get_longest_contour(contourValue=-1500., author=author)
```

Comment:
@alicebarthel, here's where the depth of the contour that splits each region gets defined.  It's -1500 m, as you specified.

Reply:
Ok, great. Thanks for clarifying!